lib/elastomer/client.rb

```diff
@@ -69,6 +69,15 @@ def info
       response.body
     end
 
+    # Returns a Tasks instance for querying the cluster bound to this client for
+    # metadata about internal tasks in flight, and to submit administrative
+    # commands concerning those tasks.
+    #
+    # Returns a new Tasks object associated with this client
+    def tasks
+      Tasks.new(self)
+    end
+
```

Comment:
Could you move this method into the `lib/elastomer/client/tasks.rb` file, please. The convention here is that an API endpoint definition includes all the methods for that endpoint. We do something similar in the `lib/elastomer/client/index.rb` file.

Reply:
Ah I see thanks will do!